Sources/Scyther/Extensions/UIImageView+Extension.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  UIImageView+Extension.swift
```

Comment:
Missing `s` on the end of the file name. Please update both the file name and the comment.